src/test/sql/sqlite_testrunner/sqlite_testrunner.cpp

```diff
@@ -24,11 +24,11 @@
 
 namespace opossum {
 
-class SQLiteTestRunner : public BaseTest {
+class SQLiteTestRunner : public testing::TestWithParam<std::string> {
  protected:
-  void _set_up() {
+  void SetUp() override {
     StorageManager::get().reset();
-    _sqlite.reset(new SQLiteWrapper());
```

Comment:
I can't believe this was in `master` without @mrks having a heart attack. Maybe `new` isnt so bad after all 🤔

Reply:
https://github.com/hyrise/hyrise/pull/412/commits/3b7f734fd0dce561911c56e6aabba7e4f247027d ;)

I think I found another regression in that PR before. Did someone already check that we didn't break anything else?

Reply:
So, I couldn't spot any other this that broke. Doesn't mean there are none of course, I only saw the one here because I was involved in the change, otherwise it might have slipped also. But yeah, anything else here, or can this be merged?